src/permission.ts

```diff
@@ -2,24 +2,22 @@ import { autoinject } from 'aurelia-dependency-injection';
 import { SessionService, SessionPort } from './session';
 
 export type PermissionType = 
-    'ar.stage'          //Geolocation
-    | 'ar.camera'       //Camera
-    | 'ar.3dmesh';      //3D Structural mesh
-
-export const PermissionNames = {
-        'ar.stage': 'Location',
-        'ar.camera': 'Camera',
-        'ar.3dmesh': 'Structural mesh'
-    };
-
-const PermissionDescriptions = {
-        'ar.stage': 'your location', 
-        'ar.camera': 'your camera',
-        'ar.3dmesh': 'the structure of your surroundings'
-    };
-/**
- * 
- */
+    'location'          //Geolocation
+    | 'camera'          //Camera
+    | 'depthmesh';      //3D Structural mesh
+
+export const EntityIdByPermissionType = {
+    'location' : ['ar.stage'],
+    'camera' : [],
+    'depthmesh' : []
+}
+
+export const PermissionTypeByEntityId = {};
+for (let type in EntityIdByPermissionType) {
+    for (let id of EntityIdByPermissionType[type])
+        PermissionTypeByEntityId[id] = type;
+}
+
```

Comment:
We don't need either of these maps if we simply check for the appropriate permission type as needed